### PR TITLE
new package: moar

### PR DESCRIPTION
**Pull request by DiegoMagdaleno**
Uncomment relevant sections and delete options which are not applicable

#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc )

### CODE REVIEW

File: srcpkgs/moar/template [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Template file for 'moar'
+pkgname=moar
+version=1.13.0
+revision=0
+build_style=go
+go_import_path=github.com/walles/moar
+hostmakedepends="git"
```
Comment (paper42) [inline srcpkgs/moar/template:7]: why is this necessary?

File: srcpkgs/moar/template [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Template file for 'moar'
+pkgname=moar
+version=1.13.0
+revision=0
```
Comment (abenson) [inline srcpkgs/moar/template:4]: `revision` should start at 1.

File: srcpkgs/moar/template [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+build_style=go
+go_import_path=github.com/walles/moar
+hostmakedepends="git"
+short_desc="Drop-in replacement for the less pager"
+maintainer="Diego Magdaleno <diegomagdaleno@pm.me>"
+license="BSD"
```
Comment (abenson) [inline srcpkgs/moar/template:10]: `BSD-2-Clause`

File: srcpkgs/moar/template [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+homepage="https://github.com/walles/moar"
+changelog="https://github.com/waller/moar/releases"
+distfiles="https://github.com/walles/moar/archive/refs/tags/v${version}.tar.gz"
+checksum=65eac0d4f8a40f0198a93f0888a60a28e8466b2ee9d0437f143db5c25d5e9319
+
+post_install() {
+  vman moar.1
```
Comment (abenson) [inline srcpkgs/moar/template:17]: Need to include the license.

`vlicense LICENSE`

**Comment by DiegoMagdaleno** [comment]
I have successfully squashed the commits and applied the suggested changes :)

**Comment by DiegoMagdaleno** [comment]
@atweiden How would you fix said issue? Adding it as a conflicts or changing the binary name?

**Comment by DiegoMagdaleno** [comment]
I think the package doesn't have issues anymore :)
